### ISSUE TITLE
NVDA keeps announcing transmission for user everytime someone new transmits on a PTT server, it's really annoying

### ISSUE THREAD

**Issue by trancer1994** [BearWare/TeamTalk5#2330]
### Description

On the latest beta for TeamTalk, NVDA which is the screenreader I use keeps announcing transmission for user, transmission for user, transmission for user everytime someone keys and then unkeys on a server which is half-duplex, so one where you have to hold down a key to speak. It's really, really annoying. I've managed to narrow down the cause and it's something in my text-to-speech settings, but I don't know which one. I don't know whether it's a bug or whether it's supposed to do this and I'm just not doing the right thing. Anyway, I want it to stop announcing that everytime someone speaks because it's really annoying and I have to keep hitting ctrl everytime it does it to shut it up. 

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

Screenreader should not announce when someone new is about to start, or finish, transmitting on a half-duplex server

### Actual behavior

Screenreader keeps announcing when someone new is transmitting on a PTT server everytime someone new speaks, specifically who the transmission is for (who it can be heard by, in other words)

### Steps to reproduce problem

1. Log onto any server which requires that you hold down a key to speak.
2. Go into a channel which is PTT-only. Two people must be in the channel, and you must have all your text-to-speech events on in the tts settings in preferences. You must be using NVDA, one or both parties. 
3. Hold down the key until you hear your key-up sound and then speak. You should hear "transmission for <user>". Then let go of the key. 
4. When the other person speaks, you should hear "transmission for you", then "transmission for everyone" while the other person speaks. Then when they are about to stop speaking, you should hear the same thing again. 
5. If you don't want to hear NVDA keep announcing that everytime someone speaks, hit the ctrl key everytime it does.

**Comment by CoBC** [comment]
Fixed in commit 5013f6d
